std/experimental/ndslice/slice.d

```diff
@@ -638,6 +638,16 @@ auto makeSlice(T,
 }
 
 /// ditto
+auto makeSlice(
+    Flag!`replaceArrayWithPointer` replaceArrayWithPointer = Yes.replaceArrayWithPointer,
```

Comment:
It's dittoed from the method above, which is part of the upcoming ndslice modifications for 2.072. [Preview here](http://dlang.org/phobos-prerelease/std_experimental_ndslice_slice.html)

@9il should we explain these flags in the slice module header?